src/main/java/org/folio/orders/utils/ErrorCodes.java

```diff
@@ -47,7 +47,10 @@
   MISSING_CONTRIBUTOR_NAME_TYPE("missingContributorNameType", "The specified contributor-name-types were not found in inventory-storage"),
   MISSING_LOAN_TYPE("missingLoanType", "Loan-type is a required field for creating an Item in Inventory"),
   ENCUMBRANCE_CREATION_FAILURE("encumbranceCreationFailure", "One or more encumbrance record(s) failed to be created"),
-  PROHIBITED_FIELD_CHANGING("protectedFieldChanging","Protected fields can't be modified");
+  PROHIBITED_FIELD_CHANGING("protectedFieldChanging","Protected fields can't be modified"),
+  UNKNOWN_USER("restrictedDueToUnknownUser", "User Id is not presented"),
+  ORDER_UNITS_NOT_FOUND("orderUnitsNotFound", "Order doesn't contain unit ids or units not found"),
```

Comment:
This error message is used only in https://github.com/folio-org/mod-orders/blob/d4eb9bb800a3885d80316013e2726a88ce44e00c/src/main/java/org/folio/rest/impl/ProtectionHelper.java#L86
and is thrown only if Acq Units are not found, so "Order doesn't contain unit ids" can be removed

Reply:
Done